### PR TITLE
Update Japanese localization on concepts/containers/images.md

### PR DESCRIPTION
**Pull request by hase1128**
**What this PR does / why we need it:**
content/ja/docs/concepts/containers/images.md is outdated.

File to update
https://github.com/kubernetes/website/tree/dev-1.19-ja.1/content/ja/docs/concepts/containers/images.md

Original
https://github.com/kubernetes/website/tree/release-1.19/content/en/docs/concepts/containers/images.md

diff between translated and v1.19
https://gist.github.com/4c810d3d117f826c0271701d1c1677ea

**Which issue(s) this PR fixes:**
#26864

### CODE REVIEW

File: content/ja/docs/concepts/containers/images.md [hunk 1]
```diff
@@ -179,15 +178,15 @@ Node構成を制御できる場合、この方法が適しています。
...
+### PodでimagePullSecretsを指定する
 
 {{< note >}}
 この方法がプライベートレジストリのイメージに基づいてコンテナを実行するための推奨の方法です。
 {{< /note >}}
 
 KubernetesはPodでのコンテナイメージレジストリキーの指定をサポートしています。
 
-#### Docker Configを利用してSecretを作成する。
+#### Docker configを利用してSecretを作成する。
```
Comment (atoato88) [inline content/ja/docs/concepts/containers/images.md:189]: The term of ["docker config"](https://docs.docker.com/engine/reference/commandline/config/) may relate to docker swarm.(I don't know the detail of this)
So is it better that translate word of `config` to `設定` ? :thinking: 
```suggestion
#### Docker の設定を利用してSecretを作成する。
```

It's LGTM except for this! :smile:

Comment (hase1128) [reply content/ja/docs/concepts/containers/images.md:189]: @atoato88 
This section does not use a unique feature called Docker config. 
This section just uses docker settings. So your suggestion makes sense.
I accept your suggestion.

**Comment by atoato88** [comment]
/lgtm
Thank you to update :smile:

**Comment by inductor** [comment]
@atoato88 Please check the issue is closed or not after merging this(if a PR already has approval and you add LGTM that means so)

https://github.com/kubernetes/website/issues/26864

**Comment by atoato88** [comment]
@inductor OK, I'll check that.
Thank you to comment.

**Pull request merged**